crates/sui-indexer/src/models/checkpoints.rs

```diff
@@ -110,13 +115,21 @@ impl Checkpoint {
                 })
             })
             .collect::<Result<Vec<TransactionDigest>, IndexerError>>()?;
+        let validator_sig = AggregateAuthoritySignature::decode_base64(&self.validator_signature)
+            .map_err(|e| {
+            IndexerError::SerdeError(format!(
+                "Failed to decode validator signature: {:?} with err: {:?}",
+                self.validator_signature, e
```

Comment:
can you add an integration test to verify that encoding / decoding of `validator_signature` actually works?

Reply:
added